bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/internal/db/JdbcPostgresqlDAO.java

```diff
@@ -153,6 +168,18 @@ public ItemsVO doCreateItemsTableIfNot(ItemsVO vo) throws JdbcSQLException {
         return vo;
     }
 
+    @Override
+    public void doDropTable(String tableName) throws JdbcSQLException {
+        String sql = StringUtilsExt.replaceArrayMerge(this.sqlDropTable, new String[] { "#tableName#" },
+                new String[] { tableName });
+        logger.debug("JDBC::doDropTable sql={}", sql);
+        try {
+            Yank.execute(sql, null);
+        } catch (YankSQLException e) {
+            throw new JdbcSQLException(e);
+        }
+    }
+
```

Comment:
It looks like this method doesn't differ from the base implementation, so there should be no reason to override it?

Reply:
The method doesn't differ but the query it uses does.

Reply:
The overridden query will still be used by the base implementation. See for example query `sqlIfTableExists` which is overridden in `JdbcPostgresqlDAO` (and other subclasses), but `doIfTableExists` is only implemented in `JdbcBaseDAO`.

Reply:
I didn't know that.